util/generate_all

```diff
@@ -6,6 +6,32 @@ if [ ! -f util/.serverpod_util_root ]; then
     exit 1
 fi
 
+generateServerUpgradeFiles() {
+    # The upgrade folder doesn't have all the files needed
+    # run `serverpod generate`. Therefore some project files have 
+    # to temporarily be copied over from the base projectname_server template.
+
+    # Save current files
+    find . -type f | sort > before.txt 
+
+    # Copy over project files temporarily from base projectname_server template
+    cp -rn $BASE/templates/serverpod_templates/projectname_server/lib/src/* lib/src/
+    cp $BASE/templates/serverpod_templates/projectname_server/pubspec.yaml pubspec.yaml
+    cp -r $BASE/templates/serverpod_templates/projectname_server/.dart_tool .
+
+    dart $CLI generate --no-analytics
+
+    # Save files after generation
+    find . -type f | sort > after.txt
+
+    # Remove all generated files except the test tools file.
+    # The generated endpoint files are already tracked in projectname_server,
+    # and should not be tracked again in the upgrade template.
+    test_tools_file="test/integration/test_tools/serverpod_test_tools.dart"
+    comm -13 before.txt after.txt | grep -v "$test_tools_file" | xargs rm 
+    rm before.txt
```

Comment:
It's already removed on line 31 because the `after.txt` includes itself, which is a diff from `before.txt`, and will therefore be deleted. `before.txt` however exists in both files and has to be explicitly deleted on line 32.